Delta/Emulation/GameViewController.swift

```diff
@@ -879,6 +890,11 @@ extension GameViewController: SaveStatesViewControllerDelegate
             {
                 try self.emulatorCore?.load(saveState)
             }
+            
+            if ExperimentalFeatures.shared.toastNotifications.stateLoadEnabled
+            {
+                self.presentExperimentalToastView(NSLocalizedString("Game State Loaded", comment: ""))
```

Comment:
[Request] Same as above, for consistency with rest of the app, we should call them Save States not Game States for now.

```suggestion
                self.presentExperimentalToastView(NSLocalizedString("Save State Loaded", comment: ""))
```

OR

```suggestion
                self.presentExperimentalToastView(NSLocalizedString("Loaded Save State", comment: ""))
```